results/intfloat__multilingual-e5-small/RomaTalesBitextMining.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Please run this evaluation again after inheriting from the CrossLingualTask class

Reply:
Will do!